base/math.jl

```diff
@@ -187,7 +187,22 @@ function evalpoly(x, p::Tuple)
     end
 end
 
-evalpoly(x, p::AbstractVector) = _evalpoly(x, p)
+function evalpoly(x, p::AbstractVector)
```

Comment:
This should probably replace `function _evalpoly(x, p)`, no?

Reply:
Yes, I just did some benchmarking for tuples and the while implementation seems to be as fast as the current `_evalpoly` version.